testsuite/src/it/java/org/graalvm/tests/integration/AppReproducersTest.java

```diff
@@ -465,10 +465,8 @@ public void imageioAWT(TestInfo testInfo, Apps app) throws IOException, Interrup
             final File executable = new File(appDir.getAbsolutePath() + File.separator + "target", "imageio");
             final Set<String> expected = Set.of("libawt.a", "libawt_headless.a", "libfdlibm.a", "libfontmanager.a", "libjava.a", "libjavajpeg.a", "libjvm.a", "liblcms.a", "liblibchelper.a", "libnet.a", "libnio.a", "libzip.a");
             if (UsedVersion.getVersion(inContainer).compareTo(Version.create(22, 2, 0)) >= 0) {
-                // libmanagement_ext.a added in 22.2 with https://github.com/oracle/graal/commit/a0e6a3aeb8b63f6c06dc3554c342075534d90796
-                // expected.add("libmanagement_ext.a");
-                // libmanagement_ext.a removed again: https://github.com/oracle/graal/pull/4383
-                // NO-OP
+                // libmanagement_ext.a added in 22.2+ with https://github.com/oracle/graal/commit/a0e6a3aeb8b63f6c06dc3554c342075534d90796
+                expected.add("libmanagement_ext.a");
             } else if (UsedVersion.jdkFeature(inContainer) > 11 || (UsedVersion.jdkFeature(inContainer) == 11 && UsedVersion.jdkUpdate(inContainer) > 12)) {
```

Comment:
The lib was added but it was not being marked as reachable by the static analysis (see https://github.com/Karm/mandrel-integration-tests/issues/109#issuecomment-1256068254). As a result this change will result in failures with 22.2. The library should only be expected to be found in versions >= 22.3